### DISCUSSION TITLE
problem base model realism engine sdxl v3.0

### DISCUSSION THREAD

**Discussion by jackalxp5b** [lllyasviel/Fooocus#3251] [Q&A] [answered]
hello to the whole community, I encountered a problem when transferring the entire fooocus folder to a second computer, with different, more powerful hardware, the software starts, and I can put back all my settings that I used on the previous computer, only that when I use the realism engine sdxl v3.0, I get the red error message... what could it be?

**Comment by IPv6** [comment]
If you are on windows - most probably you need to install fresh fooocus on new PC using official instructions - and only then copy models into models folder on new PC. On windows some ml libs are installed according to local hardware specs, you can`t just move embedded python and expect it to always run - although it may work for some pure python libs

**Comment by jackalxp5b** [comment]
yes, and in fact I installed everything from scratch... I always re-downloaded the realism engine v3, but it still doesn't work...

**Comment by jackalxp5b** [comment]
i have a gtx1080...

**Comment by ILikeToasters** [comment]
What is displayed in the CMD Window?

**Comment by jackalxp5b** [comment]
![IMG_4813 (1)](https://github.com/lllyasviel/Fooocus/assets/72381224/0cf8e7c4-3a1d-4da3-be3e-a83985766d6d)
this...

**Comment by jackalxp5b** [comment]
![IMG_4814](https://github.com/lllyasviel/Fooocus/assets/72381224/06484095-56b1-4441-b7e6-235840665382)
and this

**Reply by ILikeToasters** [reply, marked as answer]
Have you checked here and made sure not a System Swap issue? https://github.com/lllyasviel/Fooocus/blob/5a71495822a11bbabf7c889eed6d9b38b261bb96/troubleshoot.md#L4

**Comment by jackalxp5b** [comment]
no...how?

**Reply by ILikeToasters** [reply]
If you are responding to my answer about system swap follow the instructions at that link. https://github.com/lllyasviel/Fooocus/blob/5a71495822a11bbabf7c889eed6d9b38b261bb96/troubleshoot.md#L4

**Comment by jackalxp5b** [comment]
In fact, I don't have the 40GB of space required for swap, I have to find a larger disk and try again, maybe that's the problem... however, the option is active... as soon as possible I'll try with a larger disk.

**Reply by jackalxp5b** [reply]
everything is ok, it was just the lack of disk space... it works now with the larger disk!
